### ISSUE TITLE
How to test locally

### ISSUE THREAD

**Issue by Rubyj** [kislyuk/domovoi#20]
Is there support to test my s3event driven lambda locally? i,.e. with a test event json file? It is not clear from the documentation.

**Comment by kislyuk** [comment]
Yes, you can test your event handler entry point with an example event and context. There is currently nothing specific in Domovoi to support this beyond the fact that you can use your event handlers as regular functions, what would you like to see?

**Comment by Rubyj** [comment]
@kislyuk I see. Is there any way to execute this behavior on the command line with a call to `domovoi`? It sounds like what you are describing would be done through pytests. I was hoping for a way to invoke the lambda to run locally on the command line.

**Comment by kislyuk** [comment]
Not currently, but I'll consider it.
